.coveragerc

```diff
@@ -0,0 +1,17 @@
+[run]
```

Comment:
@thomasrockhu Yes this was just to check coverage manually. What happens on the main app is circleci runs the tests with coverage and uploads the results to circlci.